Resources/Prototypes/Nuclear14/Entities/Objects/Weapons/Guns/Rifles/rifles.yml

```diff
@@ -43,3 +43,141 @@
     steps: 1
     zeroVisible: true
   - type: Appearance
+
+- type: entity
+  name: Assault Carbine
+  parent: BaseWeaponRifle
+  id: WeaponCarbine
+  description: A black compact assault rifle, uses 5.56 ammo.
+  components:
+  - type: Sprite
+    sprite: Nuclear14/Objects/Weapons/Guns/Rifles/Assault Carbine.rsi
```

Comment:
Remove the spaces from the file names as it causes the tests to flip. `AssaultCarbine.rsi` will do nicely.